app/cdap/components/DirectiveInput/Components/InputPanel/index.tsx

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Typography } from '@material-ui/core';
+import MyDataPrepApi from 'api/dataprep';
+import { defaultFuseOptions } from 'components/DirectiveInput/constants';
+import { IDirectiveUsage, IObject } from 'components/DirectiveInput/types';
+import ee from 'event-emitter';
+import Fuse from 'fuse.js';
+import reverse from 'lodash/reverse';
+import Mousetrap from 'mousetrap';
+import React, { useEffect, useState } from 'react';
+import globalEvents from 'services/global-events';
+import NamespaceStore from 'services/NamespaceStore';
+import styled from 'styled-components';
+import uuidV4 from 'uuid/v4';
+import { grey } from '@material-ui/core/colors';
+import { IHeaderNamesList } from 'components/GridTable/types';
+
+interface IInputPanelProps {
+  setDirectivesList: React.Dispatch<React.SetStateAction<[]>>;
+  directiveSelected: boolean;
+  columnNamesList: IHeaderNamesList[];
+  onSearchItemClicked: (value: string) => void;
+  getDirectiveSyntax: (results: IDirectiveUsage[], value: boolean) => void;
+  onColumnSelected: (value: boolean) => void;
+  inputBoxValue: string;
+}
+
+const SimpleWrapper = styled(Box)`
+  display: block;
+`;
+
+const SmallLabel = styled(Typography)`
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  letter-spacing: 0.15;
+  color: ${grey[700]};
+`;
+
+const LargeLabel = styled(SmallLabel)`
+  font-weight: 600;
+  font-size: 16px;
+`;
+
+const ResultRow = styled(Box)`
+  padding: 10px;
+  border-bottom: 1px solid ${grey[300]};
+  background-color: ${(props) => (props.isActive ? '#EFF0F2' : '#FFFFFF')};
```

Comment:
dont use props
instead write 2 variations plz
keep common styles in ResultRow
create 2 more styled components with 2 background colors
and use as required